neuralmonkey/decoders/sequence_classifier.py

```diff
@@ -24,18 +24,49 @@ def __init__(self,
                  encoders: List[Any],
                  vocabulary: Vocabulary,
                  data_id: str,
-                 layers: Optional[List[int]]=None,
-                 activation: Callable[[tf.Tensor], tf.Tensor]=tf.tanh,
+                 layers: List[int]=None,
+                 activation: str='RELU',
```

Comment:
Protoze to neni konzistentni se zbytkem opice (viz prepinadlo GRU a LSTM) a lidi se museji dlabat v tensorflow, tohle prepinadlo mi prijde lepsi. Ale jestli tam chcete mit callabale tak se hadat nebudu je to celkem prkotina, me to takhle prijde cistsi